tests/data/datasets/test_image_segmentation.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from pathlib import Path
+
+import torch
+from albumentations.pytorch import ToTensorV2
+from albumentations.core.composition import Compose
+from albumentations import Resize, HorizontalFlip, VerticalFlip
+
+from src.data.datasets.segmentation.image_segmentation import ImageSegmentationDataset
+
+
+class TestClassificationMulticlass(unittest.TestCase):
+    def __init__(self, methodName: str = None) -> None:
+        super().__init__(methodName)
+        self.__root_dir = Path(__file__).parent
+        self.__data_folder = self.__root_dir / 'data/segmentation_data/'
+        self.__csv_path = self.__root_dir / 'data/segmentation_data/segmentation_test.csv'
+        self.__transform = Compose([Resize(224, 224), ToTensorV2()], p=1.0)
+        self.__augment = Compose([HorizontalFlip(p=0.5), VerticalFlip(p=0.5)])
+
+    def test_len(self):
```

Comment:
The tests are partially the same as for other datasets. Create a general unit test for inheritance